langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -317,6 +457,13 @@ public isolated function children(xml x) returns xml = @java:Method {
 #
 # This is equivalent to `children(x).elements(nm)`.
 #
+# ```ballerina
+# xml books = xml `<book><play>Hamlet</play></book><book><novel>Macbeth</novel></book>`;
+# books.elementChildren("novel") ⇒ <novel>Macbeth</novel>
+# 
+# books.elementChildren() ⇒ <play>Hamlet</play><novel>Macbeth</novel>
+# ```
```

Comment:
Shall we make the example without the argument the first example?

Reply:
Addressed in #39173